src/pineko/check.py

```diff
@@ -131,12 +153,7 @@ def is_fonll_b(fns, lumi):
     bool
         true if the fktable is a FONLL-B DIS fktable
     """
-    for lists in lumi:
-        for el in lists:
-            if (not islepton(el[0])) and (not islepton(el[1])):
-                # in this case we are sure it is not DIS so for sure it is not FONLL-B
-                return False
-    if fns == "FONLL-B":
+    if is_dis(lumi) and fns == "FONLL-B":
```

Comment:
I wonder if we still need an explicit check for FONLL-B @felixhekhorn 

For sure, we need to map FONLL-A/B/C on the correct massive (FO) and massless (resummed) order, but we might just need something like:
```py
fonll_schemes = {"A": (1, 1), "B": (1, 2), "C": (2, 2)}
```

Reply:
I guess with the new strategy of @andreab1997 the answer is no (since now there is an explicit spellout)